src/index.js

```diff
@@ -1 +1,56 @@
+import { url } from 'css-tree';
 import './index.css';
+
+const btn = document.querySelector('.btn-click');
+const form = document.getElementById('form');
+
+const userdatails = document.getElementById('user-details');
+const apiKey = 'N8J5KyuIIF3RfCrtF386'; // This is the Key for your game from Curl Command
+
+url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${apiKey}/scores/`;
+
+btn.addEventListener('click', () => {
+  fetch(url,
+    {
+      method: 'GET',
+    })
+    .then((res) => res.json())
+    .then((data) => {
+      userdatails.innerHTML = '';
+      data.result.forEach((el) => {
+        userdatails.innerHTML += `
+      <div class="my-div">${el.user}:${el.score}</div>`;
+      });
+      const myDiv = document.getElementsByClassName('my-div');
+      for (let i = 0; i < myDiv.length; i += 1) {
+        if (i % 2 === 0) {
+          myDiv[i].classList.add('active');
+        }
+      }
+    });
```

Comment:
- [ ] As part of the requirements you need to use async and await with the fetch method to send and receive data , kindly fix it 🙏